modules/core/src/main/scala/sangria/macros/AstLiftable.scala

```diff
@@ -142,12 +142,12 @@ trait AstLiftable {
   }
 
   implicit def liftDocument: Liftable[Document] = Liftable { case doc @ Document(d, c, p, _) =>
-    q"_root_.sangria.ast.Document($d, $c, $p, _root_.scala.Some(new _root_.sangria.ast.DefaultSourceMapper(${doc.sourceMapper.get.id}, _root_.org.parboiled2.ParserInput(${doc.source.get}))))"
+    q"_root_.sangria.ast.Document($d, $c, $p, _root_.scala.Some(new _root_.sangria.parser.DefaultSourceMapper(${doc.sourceMapper.get.id}, _root_.org.parboiled2.ParserInput(${doc.source.get}))))"
```

Comment:
What I forgot is that we need to use `SourceMapperInput` here also.
I'm split between having a helper method converting `org.parboiled2.ParserInput` to `SourceMapperInput`.
Or making `SourceMapperInput` using implicit to find the right instance so that we "only" need to import the right package.

Reply:
I see now that you did that here: https://github.com/sangria-graphql/sangria/pull/777/commits/ed755de5dfe3e4d286a18a4b921b965a9b55d0c9
👍